src/documents/IoCAPI.md

```diff
@@ -0,0 +1,133 @@
+---
+title: IoC API
+layout: default
+category: Infusion
+---
+
+# Infusion IoC API
```

Comment:
The Markdown files shouldn't have a level one header explicitly. The text in the "title" attribute above will be used as the level one header for the page in the rendered HTML.